cdap-ui/app/cdap/components/WizardModal/WizardModal.scss

```diff
@@ -16,15 +16,19 @@
 
 @import '../../styles/variables.scss';
 $wizard-header-color: #464a57;
-$wizard-modal-margin-top: 180px;
+$wizard-modal-height: 85vh;
+$wizard-modal-width: 60vw;
+$wizard-modal-margin-top: 12.5vh;
+$wizard-modal-body-max-height: 360px;
+$wizard-modal-max-width: 1050px;
 
 .wizard-modal {
   overflow-x: hidden;
   &.modal-dialog {
-    height: 65vh;
-    width: 60vw;
+    height: $wizard-modal-height;
```

Comment:
I think it should be `height`, since if we change it to `max-height` then the height of the modal will be determined by the height of its child elements, which will cause the modal to not have the desired shape.

Also, because we're using `vh` units here it would already be dynamic.